examples/annotation_label_studio/pipelines/continuous_training_pipeline.py

```diff
@@ -0,0 +1,62 @@
+#  Copyright (c) ZenML GmbH 2022. All Rights Reserved.
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at:
+#
+#       https://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+#  or implied. See the License for the specific language governing
+#  permissions and limitations under the License.
+
+from zenml.logger import get_logger
+from zenml.pipelines import pipeline
+
+LOCAL_IMAGE_FILES = "./assets/images/"
+IMAGE_REGEX_FILTER = ".*(jpe?g|png)"
+
+logger = get_logger(__name__)
+
+
+def is_cat(x):
+    # NEEDED FOR FASTAI MODEL IMPORT (when / if we use it)
+    # return labels[x] == "cat"
+    return True
```

Comment:
This might be necessary but why is it in this file?

Reply:
It needs to be in the same namespace. This stuff will be cleared up in any case when I finish the actual example.